metadata.mk

```diff
@@ -3,7 +3,7 @@
 #################################################################################################
 
 # The version of github.com/projectcalico/go-build to use.
-GO_BUILD_VER=v0.91
+GO_BUILD_VER=protoc
```

Comment:
- [ ] We will revert this change once https://github.com/projectcalico/go-build/pull/574 is merged.